src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -1,7 +1,8 @@
 package frc.robot.subsystems.gpm;
 
-import com.revrobotics.CANSparkFlex;
 import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
```

Comment:
I thought this was a neo vortex with a SparkFlex controller.

Reply:
yes

Reply:
done